helm/cluster-api/templates/crd-install/crd-job.yaml

```diff
@@ -52,6 +55,13 @@ spec:
           for deployment in $(kubectl get deployment -l app.kubernetes.io/name=cluster-operator -o name); do
             kubectl rollout restart -n giantswarm "$deployment"
           done
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          seccompProfile:
+            type: RuntimeDefault
```

Comment:
This should be set either on pod or on container level, but it doesn't need to be set on the both of them.